assets/src/js/plugins/modal.js

```diff
@@ -255,10 +259,45 @@ class GiveConfirmModal extends GiveModal {
 	}
 }
 
+/**
+ * This class will handle Form modal
+ *
+ * @since 2.1.0
```

Comment:
@Sidsector9 update `@since` tag to `2.2.0` instead of `2.1.0`. Check others places too